frontend/src/routes/Dashboard/index.js

```diff
@@ -59,63 +58,83 @@ function Dashboard(props) {
       .then((result) => setData(result));
   }, []);
 
+  const userStatus = () => (
+    <div className={classNames(styles.statusList)}>
+      <div className={classNames("row", styles.statusItem)}>
+        <span
+          className={styles.dot}
+          style={{ background: statusMapping[story.sick].color }}
+        />
+        {statusMapping[story.sick].name.toUpperCase()}
+      </div>
+      <div className={classNames("row", styles.statusItem)}>
+        <span
+          className={styles.dot}
+          style={{ background: statusMapping[story.tested].color }}
+        />
+        {statusMapping[story.tested].name.toUpperCase()}
+      </div>
+      <div></div>
+    </div>
+  );
+
+  const latestUpdate = () => (
+    <div>
+      <h3>LATEST TOTALS</h3>
+      <div className="row">
+        <div className={classNames(styles.totalItem)}>
+          ACTIVES
+          <div className={classNames(styles.totalItemNum)}>
+            {data.confirmed && data.confirmed.toLocaleString()}
+          </div>
+        </div>
+        <div className={classNames(styles.totalItem)}>
+          DEATHS
+          <div className={classNames(styles.totalItemNum)}>
+            {data.deaths && data.deaths.toLocaleString()}
+          </div>
+        </div>
+        <div className={classNames(styles.totalItem)}>
+          RECOVERED
+          <div className={classNames(styles.totalItemNum)}>
+            {data.recovered && data.recovered.toLocaleString()}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+
+  const suggestions = () => (
+    <div>
```

Comment:
Same as before here: this `div` isn't really necessary, I'd change it for an empty JSX parent `<>`